about.html

```diff
@@ -0,0 +1,133 @@
+          on this occasion, resuming our commitment, we are holding this great
+          event to find out what is new that these greats bring us chef and foodies.
+          In this conference you will be able to see the new avant-garde
+          techniques of recent times, meet and talk with important people in the culinary sector,
+          and see live preparations of new launch menus</p>
+        <p class="date-event">Please contact us per Email for any futher questions about Chef conference 2022!</p>
+        <p class="last-text">chef-conference@perubrand.pe</p>
+      </section>
+    </header>
+
+    <main>
+      <section>
+        <h2>Peru brand Logo</h2>
+
+        <div class="mid-part"> 
+          <p>
+            The Peru brand logo is designed by the best sculptors of our Peruvian highlands<br>, chosen through a contest, they
+            gave us their best ideas and creations to be able <br> to make a unique design that will last over time, in this way
+            we demonstrate our logo to <br> the world with admiration.
+          </p>
+          <a><img src="img/chef.png" alt="Chef logo" class="chef"></a>
+        </div>
+    </section>  
+    </main>
+
+    <section>
+      <h2>See the past Peru events</h2>
+      <p class="places">Take a look at the last two Chef conference which took place in Lima and Arequipa.</p>
+      <section class="back-about">  
+          <div class="mid-part"> 
+            
+            <div class="chef-event-1">
+              <div class="layer">
+                <p class="year">2019</p>
+                <p>Conference held in Lima</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="mid-part"> 
+            
+            <div class="chef-event-2">
+              <div class="layer">
+                <p class="year">2018</p>
+                <p>Conference held in Arequipa</p>
+              </div>
+            </div>
+          </div>
+      </section>
+    </section>
+
+    <section class="partner">
+      <h3>Partner</h3>
+      <div class="partner-list">
+        <p class="p1">
+          <img src="./img/coke.png" class="img-fluid" alt="coke">
+        </p>
+        <p class="p2">
+          <img src="./img/corona.png" class="img-fluid" alt="bread">
+        </p>
+        <p class="p3">
+          <img src="./img/gatorade.png" class="img-fluid" alt="food car">
+        </p>
+        <p class="p4">
+          <img src="./img/nestle.png" class="img-fluid" alt="coke2">
+        </p>
+        <p class="p5">
+          <img src="./img/heineken.png" class="img-fluid" alt="sushi">
+        </p>
+      </div>
+    </section>    
+
+    <footer>
+      <div class="footer-list">
+        <p class="copy">&copy; Peru brand</p>
+        <p class="peru">2021 Peru Company</p>
+        <p class="rights">Some Rights Reserved</p>
+      </div>
+    </footer>
+
+
+
+    <script src="main.js"></script>
```

Comment:
- [ ] I think you can use the `defer` attribute for loading your script. This attribute will make sure that your script is run only when the HTML document has finished loading. [You can read more on this here.](https://www.w3schools.com/tags/att_script_defer.asp)